sim/pokemon.ts

```diff
@@ -18,6 +18,20 @@ interface MoveSlot {
 	virtual?: boolean;
 }
 
+ /** A Pokemon's move slot as sent to the client. */
+interface RequestMoveSlot {
+	id: string;
+	move: string;
```

Comment:
Do we send `id`? It's a bit redundant with `move`, isn't it?

Reply:
Yes. It is redundant, but it's still used by the client, so we'd need a fixed client deployed before changing this.